spec/classroom_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './classroom'
+
+describe Classroom do
+  before :each do
+    @classroom = Classroom.new 'Label'
+  end
```

Comment:
- [ ] Let ensure that there are unit tests for all the methods in all the classes, please kindly write unit test for the `add_student` method in the Classroom class.